packages/desktop-client/src/components/modals/EditRule.js

```diff
@@ -227,6 +228,29 @@ function ConditionEditor({
           onDelete={isSchedule && field === 'date' ? null : onDelete}
         />
       </Stack>
+      {(field === 'account' ||
+        field === 'payee' ||
+        field === 'notes' ||
+        field === 'category' ||
+        field === 'imported_payee') && (
+        <View
+          style={{
+            flexDirection: 'row',
+            alignItems: 'center',
+            userSelect: 'none',
+          }}
+        >
+          <Checkbox
+            id="add_negate"
+            checked={selectNegate}
+            onChange={() => {
+              selectNegate = !selectNegate;
+              onChange('negate', selectNegate);
+            }}
+          />
+          <label htmlFor="add_negate">Not</label>
+        </View>
+      )}
```

Comment:
This UI feels a bit out of place. How about instead moving it to the operator menu (where it would show up as another set of operators below a divider line, with names like “is not,” “does not contain,” and “not one of?” (I don’t feel like that UI would work as well for the filter picker, but maybe it could work there too somehow for consistency?)